src/autoscaler/metricsgateway/nozzle_test.go

```diff
@@ -197,7 +197,8 @@ var _ = Describe("Nozzle", func() {
 				LogServerName = "wrong-server-name"
 			})
 			It("should output the err", func() {
-				Eventually(logger.Buffer).Should(Say("certificate is valid for reverselogproxy, not wrong-server-name"))
+				// Eventually(logger.Buffer).Should(Say("certificate is valid for reverselogproxy, not wrong-server-name"))
+				Eventually(logger.Buffer).Should(Say("Error connecting to Logs Provider"))
```

Comment:
Just out of curiosity: Why did this change? I suppose "... is valid for ...,not ..." comes from the standard library and we now check for the error that is generated by `go-loggregator`. Did they modify the error message?

Reply:
Yeah, it looks like the error coming from `go-loggregator` is coming from https://github.com/cloudfoundry/go-loggregator/blob/5adba004bd0139991d9e89899c0ceeb92db75054/envelope_stream_connector.go#L197, the older version which we using before (although I not 100% which version is was) seemed to use a `v2.NewClient` to connect which is why we may have got the error message directly.

Reply:
I do think there is a bug in the `go-loggregator` code though, as the cause of the error is not passed in the message (although it looks like it should be), I'm going to try and update to the /v8 version in a later PR that fixes the incompatible imports

Reply:
Sounds like a plan! :+1: